client/go/outline/config/module.go

```diff
@@ -99,6 +99,8 @@ func NewDefaultTransportProvider(tcpDialer transport.StreamDialer, udpDialer tra
 
 	// Stream dialers.
 	streamDialers.RegisterSubParser("shadowsocks", NewShadowsocksStreamDialerSubParser(streamEndpoints.Parse))
+	streamDialers.RegisterSubParser("direct", NewDirectStreamDialerSubParser(streamEndpoints.Parse))
```

Comment:
Instead of "direct", let's use "default", and return the default `tcpDialer` passed in the parameters. That way we can change the default if we want to test multihop.